.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/eslint-recommended"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 2018
+    },
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "rules": {
+        "no-unused-vars": "off"
```

Comment:
The linter was complaining about the unused variable `e` at https://github.com/oracc/vscode-language-sample/blob/17f63caa78df9a42cdc558121dbb45d02ab5fc2f/src/extension.ts#L100  Not sure how to fix it right now, I think we can ignore this error for the time being just for the sake of setting this up :slightly_smiling_face:

Reply:
Fixed, somehow :slightly_smiling_face: